Royalty/Patches/ThingDefs_Misc/Apparel_Armor.xml

```diff
@@ -266,7 +280,36 @@
             <JumpRange>30</JumpRange>
         </value>
     </Operation>
-
+	
+	<Operation Class="PatchOperationReplace">
+		<xpath>Defs/ThingDef[defName="Apparel_ArmorLocust"]/costList/Plasteel</xpath>
+		<value>
+			<Plasteel>220</Plasteel>
+		</value>
+	</Operation>
+	
+	<!-- ========== Prestige Armor ========== -->
```

Comment:
I don't think the prestige armor is any better stat-wise than the base armor, save a psychic boost? Not enough to justify a higher construction cost, I'd say.

Reply:
Vanilla stuff. Dunno why, but there's an extra plasteel cost associated with prestiege armor.

Reply:
Weird. Alright, well in vanilla the cost is 20% more plasteel, but I wouldn't make it that much for CE. I'd just do like 5 - 10%, depending on the final cost--a fairly token amount.